src/server/game/DataStores/DBCEnums.h

```diff
@@ -664,6 +664,62 @@ enum class ChrCustomizationReqFlag : int32
 
 DEFINE_ENUM_FLAG(ChrCustomizationReqFlag);
 
+enum CurrencyConsts
+{
+    CURRENCY_TYPE_ANCIENT_MANA    = 1155,
+    CURRENCY_TYPE_AZERITE         = 1553,
+
+    CURRENCY_MAX_CAP_ANCIENT_MANA = 2000
+};
+
+enum CurrencyTypesFlags : uint32
```

Comment:
```suggestion
enum class CurrencyTypesFlags : uint32
```
That way you can remove the Currency* prefixes from enum entries

Reply:
Thanks! Updated the enum to enum class

I've taken the names as they appear here: https://github.com/tomrus88/BlizzardInterfaceCode/blob/master/Interface/AddOns/Blizzard_APIDocumentationGenerated/CurrencyConstantsDocumentation.lua (using Currency* prefix)